var/spack/repos/builtin/packages/enzyme/package.py

```diff
@@ -23,6 +23,7 @@ class Enzyme(CMakePackage):
     root_cmakelists_dir = "enzyme"
 
     version("main", branch="main")
+    version("0.0.172", sha256="688200164787d543641cb446cff20f6a8e8b5c92bb7032ebe7f867efa67ceafb")
```

Comment:
Do I need to modify the LLVM dependency requirements, e.g., to `llvm@15:19`?
https://github.com/EnzymeAD/Enzyme/pull/2074

Reply:
yeah we dropped LLVM before 15

Reply:
bumped.
https://github.com/EnzymeAD/Enzyme/releases/tag/v0.0.149